Cargo.toml

```diff
@@ -27,7 +27,7 @@ clock = [
   "dep:android-tzdata",
   "now",
 ]
-now = ["std"]
+now = []
```

Comment:
So I think we should revert this change, because `now` without `std` is pretty much meaningless.

Reply:
I think `now` should enable calling `Utc::now()` on most common platforms. If it doesn't, that feels broken.

Reply:
It does. https://github.com/chronotope/chrono/pull/1552/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R474

Reply:
It seems surprising that the `cfg(feature = "std")` guard doesn't match the behavior of whether we enable/disable `no_std` exactly (for the library).

Reply:
It's a choice we can make. Separating the features allows things like using `Utc::now` in combination with `wasmbind` without depending on the standard library.

Reply:
Okay, let's try it. Easier to add it later than to remove it.